docker/vm_startup_script.sh

```diff
@@ -1,3 +0,0 @@
-#! /bin/bash
-# Delete old catalystcoop/pudl-etl images
-docker rmi -f $(docker images -q catalystcoop/pudl-etl)
```

Comment:
Is this no longer necessary? Or do we still need it while we're not on Batch?

Reply:
I just didn't see the script being called anywhere and saw no documentation as to its being needed so I figured it was another stale leftover script from our local docker-compose setup.

Reply:
Added this script back.